tools/scrape/vimorg.py

```diff
@@ -23,25 +24,67 @@ def __init__(self, *args, **kwargs):
 PARSER = HTMLParser()
 
 
-def scrape_scripts(num):
-    """Scrapes and upserts the num most recently created scripts on vim.org."""
+def get_all_info_from_url_and_name(vimorg_url, name):
+    vimorg_data = urlparse.urlparse(vimorg_url)
+    query_string = urlparse.parse_qs(vimorg_data.query)
+    if not query_string or 'script_id' not in query_string:
+        return {}
+
+    vimorg_id = query_string['script_id'][0]
     res = requests.get(
-            'http://www.vim.org/scripts/script_search_results.php?show_me=%d' %
-                num)
+        'http://www.vim.org/scripts/script_search_results.php?keywords=%s' %
```

Comment:
I wonder if there are any cases in which the search doesn't turn up the plugin, e.g. maybe if the name was changed ...

Reply:
I think it can happen. In that case, this will not return anything.